project/Build.scala

```diff
@@ -1216,8 +1226,50 @@ object Build extends sbt.Build {
     base = file("test-suite/js"),
     settings = commonSettings ++ myScalaJSSettings ++ testTagSettings ++
       testSuiteCommonSettings(isJSTest = true) ++ Seq(
-        name := "Scala.js test suite",
+        name := "Scala.js test suite"
+      ) ++ inConfig(Test)(Seq(
+        artifactPath in jsTestDefinitions :=
+          crossTarget.value / "test_definitions.js",
```

Comment:
Nitpick: use `-` instead of `_`. AFAIK we've always used `-` in file names so far, and never `_`.